docs/contributing/issue_tracking.rst

```diff
@@ -10,31 +10,38 @@ An issue must always correspond to a specific action with a well-defined complet
 
 Do not use issues for support queries or other questions ("How do I do X?" - although "Implement a way of doing X" or "Document how to do X" could well be valid issues). These should be asked on `Stack Overflow <http://stackoverflow.com/questions/tagged/wagtail>`_, or for discussions that do not fit Stack Overflow's question-and-answer format, the `Wagtail Support Google group <https://groups.google.com/forum/#!forum/wagtail>`_.
 
-As soon as a ticket is opened - ideally within one day - a member of the core team will give it an initial classification, by either closing it as invalid or assigning it to a milestone. Don't be discouraged if you feel that your ticket has been given a lower priority than it deserves - this decision isn't permanent. We will consider all feedback, and reassign or reopen tickets where appropriate.
+As soon as a ticket is opened - ideally within one day - any member of the core team can give it an initial classification, by either closing it as invalid or assigning it to a milestone. Don't be discouraged if you feel that your ticket has been given a lower priority than it deserves - this decision isn't permanent. We will consider all feedback, and reassign or reopen tickets where appropriate.
```

Comment:
I did notice that, and honestly found it very confusing. This is all about issue triage, which in my opinion would be more understandable for more audiences if it wasn't written from any perspective.

This isn't was this PR is about though so I've just reverted those changes 🙂.